default/resident/channels.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "name"    : "activation",
+        "type"    : "EMAIL",
+        "address" : "ace_de@oracle.com"
+    },
+    {
+        "name"    : "events",
+        "type"    : "EMAIL",
+        "address" : "${owner}"
+    },
+    {
+        "name"    : "incident",
+        "type"    : "SLACK",
+        "address" : "https://bit.ly/3iqR5H8"
```

Comment:
Probably should sanitize this too - provide a placeholder that requires user input before it's usable.

Reply:
The dummy is needed to avoid error messages. This channel does exist, but I hide it behind a bitly short cut.